### ISSUE TITLE
AndroidRuntimeException requestFeature() must be called before adding content

### ISSUE THREAD

**Issue by jchristof** [mParticle/mparticle-android-sdk#36]
These crash types are showing up in Firebase Crashlytics for us. These are typically Android 6.0.1 devices. mparticle version 5.9.2.

```
Caused by android.util.AndroidRuntimeException: requestFeature() must be called before adding content
       at com.android.internal.policy.PhoneWindow.requestFeature + 398(PhoneWindow.java:398)
       at android.app.Activity.requestWindowFeature + 3993(Activity.java:3993)
       at com.appboy.ui.AppboyWebViewActivity.onCreate + 38(AppboyWebViewActivity.java:38)
       at android.app.Activity.performCreate + 6876(Activity.java:6876)
       at android.app.Instrumentation.callActivityOnCreate + 1135(Instrumentation.java:1135)
       at android.app.ActivityThread.performLaunchActivity + 3206(ActivityThread.java:3206)
       at android.app.ActivityThread.handleLaunchActivity + 3349(ActivityThread.java:3349)
       at android.app.ActivityThread.access$1100 + 221(ActivityThread.java:221)
       at android.app.ActivityThread$H.handleMessage + 1794(ActivityThread.java:1794)
       at android.os.Handler.dispatchMessage + 102(Handler.java:102)
       at android.os.Looper.loop + 158(Looper.java:158)
       at android.app.ActivityThread.main + 7224(ActivityThread.java:7224)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run + 1230(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main + 1120(ZygoteInit.java:1120)
```

**Comment by samdozor** [comment]
hey @jchristof - this looks to be an issue within the Braze SDK. Though you're on an older mParticle and Braze SDK version, I took a look at their class `AppboyWebViewActivity` and I don't see any changes that they've made to it that would have resolved this as of their latest version. I'll keep this issue open for now but please ping your Braze representative, and I'll do the same.

**Comment by samdozor** [comment]
hey @jchristof Braze has been notified. I'm going to close this issue as it's specific to their SDK. We will look out for their updated SDK, and/or you can override our dependency at any time.

**Comment by radixdev** [comment]
This will be fixed in the next version of the Braze Android SDK

**Comment by radixdev** [comment]
This is fixed in [3.7.1](https://github.com/Appboy/appboy-android-sdk/compare/v3.7.0...v3.7.1) of the Braze Android SDK.

**Comment by jchristof** [comment]
Thanks - @samdozor we use Braze via mparticle. Is there an eta for that release? Thanks!
